hw02-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,32 @@
+package homework;
+
+import java.util.*;
+
+public class CustomerService {
+
+    TreeMap<Customer, String> customersMap = new TreeMap<>(Comparator.comparingLong(Customer::getScores));
```

Comment:
1) Не забывайте `private` для полей.
2) Тип лучше объявлять интерфейсом.
```suggestion
    private NavigableMap<Customer, String> customersMap = new TreeMap<>(Comparator.comparingLong(Customer::getScores));
```

Reply:
Согласен, не обратил на это внимания. Исправлено